[BUG] ut failed in thinRuntime

**What is your environment(Kubernetes version, Fluid version, etc.)**

**Describe the bug**

```
--- FAIL: TestThinEngine_transformFuse (0.01s)
    --- FAIL: TestThinEngine_transformFuse/test (0.01s)
        deepequal.go:27: 
            	Error Trace:	/home/runner/work/fluid/fluid/src/github.com/fluid-cloudnative/fluid/pkg/ddc/thin/deepequal.go:27
            	            				/home/runner/work/fluid/fluid/src/github.com/fluid-cloudnative/fluid/pkg/ddc/thin/deepequal.go:35
            	            				/home/runner/work/fluid/fluid/src/github.com/fluid-cloudnative/fluid/pkg/ddc/thin/transform_fuse_test.go:517
            	Error:      	elements differ
            	            	
            	            	extra elements in list A:
            	            	([]interface {}) (len=1) {
            	            	 (v1.EnvVar) {
            	            	  Name: (string) (len=13) "MOUNT_OPTIONS",
            	            	  Value: (string) (len=7) "a=b,c=d",
            	            	  ValueFrom: (*v1.EnvVarSource)(<nil>)
            	            	 }
            	            	}
            	            	
            	            	
            	            	extra elements in list B:
            	            	([]interface {}) (len=1) {
            	            	 (v1.EnvVar) {
            	            	  Name: (string) (len=13) "MOUNT_OPTIONS",
            	            	  Value: (string) (len=7) "c=d,a=b",
            	            	  ValueFrom: (*v1.EnvVarSource)(<nil>)
            	            	 }
            	            	}
            	            	
            	            	
            	            	listA:
            	            	([]v1.EnvVar) (len=4) {
            	            	 (v1.EnvVar) {
            	            	  Name: (string) (len=1) "a",
            	            	  Value: (string) (len=1) "b",
            	            	  ValueFrom: (*v1.EnvVarSource)(<nil>)
            	            	 },
            	            	 (v1.EnvVar) {
            	            	  Name: (string) (len=1) "b",
            	            	  Value: (string) "",
            	            	  ValueFrom: (*v1.EnvVarSource)({
            	            	   FieldRef: (*v1.ObjectFieldSelector)(<nil>),
            	            	   ResourceFieldRef: (*v1.ResourceFieldSelector)(<nil>),
            	            	   ConfigMapKeyRef: (*v1.ConfigMapKeySelector)({
            	            	    LocalObjectReference: (v1.LocalObjectReference) {
            	            	     Name: (string) (len=7) "test-cm"
            	            	    },
            	            	    Key: (string) "",
            	            	    Optional: (*bool)(<nil>)
            	            	   }),
            	            	   SecretKeyRef: (*v1.SecretKeySelector)(<nil>)
            	            	  })
            	            	 },
            	            	 (v1.EnvVar) {
            	            	  Name: (string) (len=11) "MOUNT_POINT",
            	            	  Value: (string) (len=26) "/thin/fluid/test/thin-fuse",
            	            	  ValueFrom: (*v1.EnvVarSource)(<nil>)
            	            	 },
            	            	 (v1.EnvVar) {
            	            	  Name: (string) (len=13) "MOUNT_OPTIONS",
            	            	  Value: (string) (len=7) "a=b,c=d",
            	            	  ValueFrom: (*v1.EnvVarSource)(<nil>)
            	            	 }
            	            	}
            	            	
            	            	
            	            	listB:
            	            	([]v1.EnvVar) (len=4) {
            	            	 (v1.EnvVar) {
            	            	  Name: (string) (len=1) "a",
            	            	  Value: (string) (len=1) "b",
            	            	  ValueFrom: (*v1.EnvVarSource)(<nil>)
            	            	 },
            	            	 (v1.EnvVar) {
            	            	  Name: (string) (len=1) "b",
            	            	  Value: (string) "",
            	            	  ValueFrom: (*v1.EnvVarSource)({
            	            	   FieldRef: (*v1.ObjectFieldSelector)(<nil>),
            	            	   ResourceFieldRef: (*v1.ResourceFieldSelector)(<nil>),
            	            	   ConfigMapKeyRef: (*v1.ConfigMapKeySelector)({
            	            	    LocalObjectReference: (v1.LocalObjectReference) {
            	            	     Name: (string) (len=7) "test-cm"
            	            	    },
            	            	    Key: (string) "",
            	            	    Optional: (*bool)(<nil>)
            	            	   }),
            	            	   SecretKeyRef: (*v1.SecretKeySelector)(<nil>)
            	            	  })
            	            	 },
            	            	 (v1.EnvVar) {
            	            	  Name: (string) (len=13) "MOUNT_OPTIONS",
            	            	  Value: (string) (len=7) "c=d,a=b",
            	            	  ValueFrom: (*v1.EnvVarSource)(<nil>)
            	            	 },
            	            	 (v1.EnvVar) {
            	            	  Name: (string) (len=11) "MOUNT_POINT",
            	            	  Value: (string) (len=26) "/thin/fluid/test/thin-fuse",
            	            	  ValueFrom: (*v1.EnvVarSource)(<nil>)
            	            	 }
            	            	}
            	Test:       	TestThinEngine_transformFuse/test
```

**What you expect to happen:**

**How to reproduce it**

**Additional Information**

--- a/pkg/ddc/thin/transform_fuse_test.go
+++ b/pkg/ddc/thin/transform_fuse_test.go
@@ -450,7 +450,7 @@ func TestThinEngine_transformFuse(t1 *testing.T) {
 				},
 			}, {
 				Name:  common.ThinFuseOptionEnvKey,
-				Value: "c=d,a=b",
+				Value: "a=b,c=d",
 			}, {
 				Name:  common.ThinFusePointEnvKey,
 				Value: "/thin/fluid/test/thin-fuse",

--- a/pkg/ddc/thin/transform_fuse_test.go
+++ b/pkg/ddc/thin/transform_fuse_test.go
@@ -514,6 +514,7 @@ func TestThinEngine_transformFuse(t1 *testing.T) {
 			t1.Errorf("transformFuse() error = %v", err)
 		}
 
+		value.Fuse.Envs = testutil.SortEnvVarByName(value.Fuse.Envs, common.ThinFuseOptionEnvKey)
 		if !testutil.DeepEqualIgnoringSliceOrder(t1, value.Fuse, wantValue.Fuse) {
 			valueYaml, _ := yaml.Marshal(value.Fuse)
 			wantYaml, _ := yaml.Marshal(wantValue.Fuse)

